bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -115,6 +115,25 @@ spec:
 
           bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
 
+          check_db_health() {
+            db="${1}"
```

Comment:
this is okay, but technically I was wrong.  

It turns out, in bash, variable assignment is the one case that doesn't need quoting, because the shell only allows a single word after the `=`

http://rg1-teaching.mpi-inf.mpg.de/unixffb-ss98/quoting-guide.html#para:sh-assignments

But always quoting seems safest rule to memorize.  It's not easy being pedantic. :laughing:

Sorry for the thrash on all this.

Reply:
Np, there's definetely room for improvement in my bash skills and I am happy to learn. Moreover, I can't say no to Cobra Commander :P